src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -111,6 +111,10 @@ object Parameters {
     "partition_size_in_mb"
   )
   val PARAM_TIME_OUTPUT_FORMAT: String = knownParam("time_output_format")
+  val PARAM_TIMESTAMP_NTZ_OUTPUT_FORMAT: String = knownParam("timestamp_ntz_output_format")
+  val PARAM_TIMESTAMP_LTZ_OUTPUT_FORMAT: String = knownParam("timestamp_ltz_output_format")
+  val PARAM_TIMESTAMP_TZ_OUTPUT_FORMAT: String = knownParam("timestamp_tz_output_format")
+  val TIMESTAMP_OUTPUT_FORMAT_SF_CURRENT: String = "sf_current"
```

Comment:
If user set these 3 options as `sf_current`, Spark connector will not set the option for the user.

If users set these 3 options and sfTimezone as `sf_current`, Spark Connector will skip the `alter session` command to set `timestamp and timezone` format.